launcher/modplatform/modpacksch/FTBPackInstallTask.cpp

```diff
@@ -210,13 +213,16 @@ void PackInstallTask::onResolveModsSucceeded()
         auto message_dialog = new BlockedModsDialog(m_parent, tr("Blocked files found"),
                                                    tr("The following files are not available for download in third party launchers.<br/>"
                                                       "You will need to manually download them and add them to the instance."),
-                                                   text,
-                                                   urls);
+                                                   m_blocked_mods);
 
-        if (message_dialog->exec() == QDialog::Accepted)
+        if (message_dialog->exec() == QDialog::Accepted) {
+            qDebug() << "Post dialog mods list: " << m_blocked_mods;
```

Comment:
Maybe this should be added to CF instance creation task. Or remove it here

Reply:
This was mostly used during the initial debugging but it seems useful enough to keep in the logs at the time. I forgot that I hadn't put it on the Flame creation task too. Would you prefer it removed or duplicated for consistency?

Reply:
I think it would be nice for debugging, so sure, go ahead and add it to Flame as well :D